src/Private/Context.ps1

```diff
@@ -339,9 +339,18 @@ function New-PodeContext {
     # sessions
     $ctx.Server.Sessions = @{}
 
-    # swagger and openapi
-    $ctx.Server.OpenAPI = Get-PodeOABaseObject
+    #OpenApi Definition Tag
+    if ((Test-PodeIsEmpty $ctx.Server.Configuration.Server.DefaultOADefinitionTag)) {
+        $ctx.Server.DefaultOADefinitionTag = 'default'
+    } else {
+        $ctx.Server.DefaultOADefinitionTag = $ctx.Server.Configuration.Server.DefaultOADefinitionTag
+    }
 
+    $ctx.Server.SelectedOADefinitionTag = $ctx.Server.DefaultOADefinitionTag
+
+    # swagger and openapi
+    $ctx.Server.OpenAPI = @{ $ctx.Server.SelectedOADefinitionTag = Get-PodeOABaseObject }
+    $ctx.Server.OADefinitionTagSelectionStack = New-Object 'System.Collections.Generic.Stack[System.Object]'
```

Comment:
I'm thinking it might be best to move all of these properties into a central ".Server.OpenAPI" space - save the main ".Server" space from getting cluttered.

(same applies to the server.psd1 config file as well)

Something like:
```powershell
#OpenApi Definition Tag
$ctx.Server.OpenAPI = @{
    DefinitionTagSelectionStack = New-Object 'System.Collections.Generic.Stack[System.Object]'
}

if ((Test-PodeIsEmpty $ctx.Server.Configuration.Server.OpenAPI.DefaultDefinitionTag)) {
    $ctx.Server.OpenAPI.DefaultDefinitionTag = 'default'
} else {
    $ctx.Server.OpenAPI.DefaultDefinitionTag = $ctx.Server.Configuration.Server.OpenAPI.DefaultDefinitionTag
}

$ctx.Server.OpenAPI.SelectedDefinitionTag = $ctx.Server.OpenAPI.DefaultDefinitionTag

# swagger and openapi
$ctx.Server.OpenAPI.Definitions = @{ $ctx.Server.OpenAPI.SelectedDefinitionTag = Get-PodeOABaseObject }
```

Reply:
done